app/controllers/oidc/pending_trusted_publishers_controller.rb

```diff
@@ -4,8 +4,8 @@ class OIDC::PendingTrustedPublishersController < ApplicationController
   before_action :find_pending_trusted_publisher, only: %i[destroy]
 
   def index
-    trusted_publishers = current_user
-      .oidc_pending_trusted_publishers.unexpired.includes(:trusted_publisher)
+    trusted_publishers = policy_scope(OIDC::PendingTrustedPublisher)
+      .unexpired.includes(:trusted_publisher)
```

Comment:
I thought about moving unexpired into the scope, but really it's not a matter of policy that you can't see expired ones, it's a matter of providing a nice interface.